src/constants/cost-estimation.ts

```diff
@@ -0,0 +1,9 @@
+// Contract Gas Price are all estimated averaged gas for a number of runs.
+export const transferHoldershipGas = 43634;
+export const transferOwnershipGas = 324688;
+export const issueDocGas = 239523;
+export const burnDocGas = 56532;
+export const surrenderDocGas = 93435;
+
+export const gweiFactor = 0.000000001;
+export const refreshRate = 30000;
```

Comment:
qn, do you know about CONSTANTS convention?

Reply:
sorry my bad, i missed that, will update.